crates/zkvm-interface/src/lib.rs

```diff
@@ -27,6 +28,7 @@ pub trait Compiler {
 
 /// ResourceType specifies what resource will be used to create the proofs.
 #[derive(Debug, Clone, Default)]
+#[cfg_attr(feature = "clap", derive(clap::Subcommand))]
```

Comment:
Re https://github.com/eth-act/ere/pull/66#discussion_r2242906289 it seems much cleaner to do the derive so applied the suggestion.